docs/source/deploy-model.md

```diff
@@ -39,7 +39,7 @@ In this way [ArgoCD][2] will be used to maintain your application always in sync
 
 Note: [AICoE Pipeline][3] can also update automatically the [imagestreamtag](https://github.com/thoth-station/elyra-aidevsecops-tutorial/blob/bb6fad2441e8df8aa56c2c0e6b5ac45a2cda42eb/manifests/overlays/test/imagestreamtag.yaml#L10) once a new release is created.
 
-Once everything is synced to the cluster, you can monitor your application from the [ArgoCD][2] using this [link](https://argocd-server-argocd.apps.cnv.massopen.cloud/applications) as shown in the image below:
+Once everything is synced to the cluster, you can monitor your application from the [ArgoCD][2] using this [link](https://argocd-server-argocd.apps.zero.massopen.cloud/applications) as shown in the image below:
```

Comment:
I think the ArgoCD service migrated too.. @harshad16

Reply:
Actually, the ArgoCD service is located on the moc-infra cluster. Please change this url to:

https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications

Reply:
yes, correct url is the one @tumido  provided: https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications